ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -0,0 +1,189 @@
+        }
+    }
+    private let avatarView: AvatarView
+    private var titleLabel: UILabel?
+    private var subtitleLabel: UILabel?
+
+    /// Initializes the contact view by creating an avatar view with a first name and last name
+    ///
+    /// - Parameters:
+    ///   - title: String that will be the text of the top label
+    ///   - subtitle: String that will be the text of the bottom label
+    @objc public convenience init(title: String, subtitle: String) {
+        self.init(title: title, subtitle: subtitle, identifier: nil)
+    }
+
+    /// Initializes the contact view by creating an avatar view with an identifier
+    ///
+    /// - Parameters:
+    ///   - identifier: String that will be used to identify the contact (e.g. email, phone number, first name)
+    @objc public convenience init(identifier: String) {
+        self.init(title: nil, subtitle: nil, identifier: identifier)
+    }
+
+    private init(title: String?, subtitle: String?, identifier: String?) {
+        avatarView = AvatarView(avatarSize: .extraExtraLarge, withBorder: false, style: .circle)
+        super.init(frame: .zero)
+
+        if let title = title, let subtitle = subtitle {
+            subtitleLabel = UILabel()
+            setupAvatarView(with: title, and: subtitle, or: nil)
+            setupTitleLabel(using: title)
+            setupSubtitleLabel(using: subtitle)
+        } else if let identifier = identifier {
+            setupAvatarView(with: nil, and: nil, or: identifier)
+            setupTitleLabel(using: identifier)
+        }
+
+        backgroundColor = Colors.surfacePrimary
+        setupLayout()
+    }
+
+    public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func setupAvatarView(with firstName: String?, and lastName: String?, or identifier: String?) {
+        if let firstName = firstName, let lastName = lastName {
+            let fullName = firstName + " " + lastName
+            avatarView.setup(primaryText: fullName, secondaryText: identifier, image: avatarImage)
+        } else {
+            avatarView.setup(primaryText: identifier, secondaryText: "", image: avatarImage)
+        }
+    }
+
+    private func setupLayout() {
+        var constraints = [NSLayoutConstraint]()
+        constraints.append(widthAnchor.constraint(equalToConstant: Constants.contactWidth))
+        constraints.append(contentsOf: avatarLayoutConstraints())
+
+        if let titleLabel = titleLabel, let subtitleLabel = subtitleLabel {
+            constraints.append(contentsOf: titleLabelLayoutConstraints())
+            constraints.append(contentsOf: subtitleLabelLayoutConstraints())
+            addSubview(titleLabel)
+            addSubview(subtitleLabel)
+        } else if let titleLabel = titleLabel {
+            constraints.append(contentsOf: identifierLayoutConstraints())
+            addSubview(titleLabel)
+        }
```

Comment:
I think this might be simpler to write as:

if let titleLabel = titleLabel {
      if let subtitleLabel = subtitleLabel {
      } else {
      }
}

Reply:
It looks much easier to read this way!